client/package.json

```diff
@@ -16,12 +16,14 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.25.0",
     "bcrypt": "^5.0.1",
+    "materialize-css": "^1.0.0",
     "multer": "^1.4.4",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.2.3",
     "react-dom": "^17.0.2",
     "react-material-ui-carousel": "^3.1.1",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "5.0.0",
```

Comment:
@ngray122 why the downgrade in version here? This is typically a no-no since most of the time you want to be using the most up to date versions of packages